components/Board/Profiles.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import Profile from "./Profile";
+
+const arr = [
+	{
+		Id: "a",
+		Name: "Alex Hsieh",
```

Comment:
As mentioned before, ensure that variable names are all lowercase and use camel casing if multiple words.